ui-v2/app/templates/dc/services/instance/meta.hbs

```diff
@@ -0,0 +1,24 @@
+<div id="meta" class="tab-section">
+  <div role="tabpanel">
+    <section class="tags">
+      <h3>Tags</h3>
+    {{#if (gt item.Tags.length 0) }}
+      <TagList @item={{item}} />
+    {{else}}
+      <p>
+        There are no tags.
+      </p>
+    {{/if}}
+    </section>
+    <section class="meta">
+      <h3>Meta</h3>
```

Comment:
I think we should use `Metadata` and `class="metadata"` here also.

Reply:
I'll change the class, but I'll leave the header as `Meta` since that's what the designs say.